package.json

```
@@ -28,7 +28,7 @@
     "jsonlint": "node scripts/jsonlint.js",
     "typescript": "lerna run typescript --parallel",
     "build:codesandbox": "yarn lerna:build",
-    "lerna:build": "cd packages/x-license && yarn build && cd ../grid/x-grid-modules && yarn build && cd ../data-grid && yarn build && cd ../x-grid && yarn build && cd ../x-grid-data-generator && yarn build",
+    "lerna:build": "cd packages/x-license && yarn build && cd ../grid && yarn build && cd ../x-grid-data-generator && yarn build",
```

Once we make x-grid not include x-license, this should work:

```suggestion
    "lerna:build": "lerna run --parallel build",
```

---

And per the exploration in https://github.com/mui-org/material-ui-x/pull/343#issuecomment-697028491, solving the x-license issue is required.